dotnet/src/SemanticKernel/Memory/MemoryRecordMetadata.cs

```diff
@@ -47,6 +47,13 @@ public class MemoryRecordMetadata : ICloneable
     [JsonPropertyName("text")]
     public string Text { get; }
 
+    /// <summary>
+    /// Field for saving custom metadata with a memory.
+    /// </summary>
+    [JsonInclude]
+    [JsonPropertyName("value_string")]
+    public string ValueString { get; }
```

Comment:
Same question: is this meant to be "the value" or just nice things we may want to know about for later? Should it be a string or a property bag (dict)?

Reply:
Commented above. It's meant to support any additional metadata that the user may want to associate with the memory. I am strongly in favor of a string - placing the responsibility of serialization/deserialization of the custom fields on the application.

Reply:
renamed to additionalMetadata